README.md

```diff
@@ -897,9 +897,9 @@ Otherwise, please add it to the [WIP section](#wip) below.
 [31]: https://blog.eldruin.com/si4703-fm-radio-receiver-driver-in-rust/
 [32]: https://blog.eldruin.com/pca9685-pwm-led-servo-controller-driver-in-rust/
 [33]: https://blog.eldruin.com/veml6030-ambient-light-sensor-driver-in-rust/
-[34]: https://ryan.kurte.nz/notes/2020-01-05-rust-radio
-[35]: https://ryan.kurte.nz/notes/2020-01-05-rust-radio
-[36]: https://ryan.kurte.nz/notes/2020-01-05-rust-radio
+[34]: http://web.archive.org/web/20230328083333/https://ryan.kurte.nz/notes/2020-01-05-rust-radio
```

Comment:
(this doesn't need to block merging), but @ryankurte is there an updated link for these you'd like us to use instead of internet archive? If so, feel free to open a follow-on PR